src/components/app/App.js

```diff
@@ -1,8 +1,55 @@
 import React from 'react';
-import { ShowContext, ShowContextProvider } from '../../contexts/ShowContext';
+import { ShowContextProvider } from '../../contexts/ShowContext';
+import { Route, Switch } from 'react-router-dom';
+import Navbar from './components/navbar/Navbar';
+import Container from './components/container/Container';
+import Footer from './components/footer/Footer';
 
 function App() {
-  return <ShowContextProvider></ShowContextProvider>;
+  return (
+    <ShowContextProvider>
+      <Switch>
+        <Route
+          exact
+          path="gone-phishin/years"
+          render={() => {
+            <div className="app">
+              <Navbar />
+              <Container type="years" />
+              <Footer />
+            </div>;
+          }}
+        />
+        <Route
+          exact
+          path="gone-phishin/years/:year"
+          render={({ match }) => {
+            return (
+              <div className="app">
+                <Navbar />
+                <Container id={match.params.year} />
+                <Footer />
+              </div>
+            );
+          }}
+        />
+        <Route
+          exact
+          path="gone-phishin/shows/:id"
+          render={({ match }) => {
+            return (
+              <div className="app">
+                <Navbar />
+                <Container id={match.params.id} />
+                <Footer />
+              </div>
+            );
+          }}
+        />
+        <Route render={() => <p>Error. Page Not Found!</p>} />
```

Comment:
Be sure to give them a way to get to a page that does exist. A button that takes them home would be great!